src/SFA.DAS.RoatpOversight.Web.UnitTests/Services/ApplicationOutcomeOrchestratorTests.cs

```diff
@@ -57,6 +57,11 @@ public void Before_each_test()
 
             _applicationApiClient.Setup(x => x.RecordOutcome(It.Is<RecordOversightOutcomeCommand>(y => y.ApplicationId == _applicationId))).ReturnsAsync(true);
 
+            _applicationApiClient
+                .Setup(x => x.RecordGatewayFailOutcome(
+                    It.Is<RecordOversightGatewayFailOutcomeCommand>(y => y.ApplicationId == _applicationId)))
+                .Returns(Task.CompletedTask);
+
```

Comment:
This feels a bit dubious as I suspect this is an Async method but the setup has been done as if it's a synchronous one

Reply:
The setup is to return a Task.